common/lib/xmodule/xmodule/tests/test_randomize_module.py

```diff
@@ -97,8 +97,8 @@ def inner_get_module(descriptor):
         randomize_module = RandomizeModule(
             randomize_descriptor,
             self.system,
-            DictFieldData({}),
-            ScopeIds(None, None, self.course.id, self.course.id)
+            field_data=None,
+            scope_ids=ScopeIds(None, None, self.course.id, self.course.id)
         )
         self.assertTrue(randomize_module.child.display_name in ['A', 'B'])
         self.assertTrue(randomize_module.get_children, 1)
```

Comment:
Why do we have that second argument `1`. Should this be: `self.assertEquals(randomize_module.get_children(), 1)`